### PR TITLE
Improve md formatting for Readme.

### PR DESCRIPTION
**Pull request by scottdangelo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,14 +67,22 @@ If you followed the previous steps you should already be inside your local clone
...
-    Make sure you have enabled USB Debugging:
-    A. Open Settings-> About-> Software Information-> More.
-    B. Then tap �Build number� seven times to enable Developer options. ...
-    C. Go back to Settings menu and now you'll be able to see �Developer options� there.
-    D. Tap it and turn on USB Debugging from the menu on the next screen.
+11. To Build an android .apk file and deploy to your phone, you can File -> Build Settings (Ctrl + Shift +B) and click Build. 
+12. When prompted you can name your build and then move it to your phone.
+13. Alternately, connect the phone via USB and File-> Build and Run (or Ctrl+B).
+
+   *Make sure you have enabled USB Debugging:*
```
Comment (markstur) [inline README.md]: this block looks OK, but it is a little strange like a code block.  bullets or sub-list might look better

Comment (markstur) [inline README.md]: I didn't understand the "move it"/"alternatively" thing. I guess there are 2 paths to take. Build or Build-and-Run. That could probably be clarified unless it is more obvious when someone really does it.

Comment (scottdangelo) [inline README.md]: fixed

File: README.md [hunk 2]
```diff
@@ -67,14 +67,22 @@ If you followed the previous steps you should already be inside your local clone
...
+
+    Then tap “Build number” seven times to enable Developer options.
+
+     Go back to Settings menu and now you'll be able to see “Developer options” there.
+
+     Tap it and turn on USB Debugging from the menu on the next screen.
+
    Once the app is deployed to your phone it will start, but you'll need to set permissions for the app before it will work correctly:
+
     A. Open Settings-> Apps-> SpeechSandboxCardboard-> Permissions and enable Microphone and Storage.
```
Comment (scottdangelo) [inline README.md]: fixed

**Pull request merged**